esp8266/scripts/main.py

```diff
@@ -0,0 +1,2 @@
+# This script is run on boot
+print("Hello!")
```

Comment:
I don't think we want it to print this by default... it's good to give an example of where to put your code, but we need a way to do it such that the default does nothing.